components/SettingsControls/src/SettingsCard/SettingsCard.xaml

```diff
@@ -439,32 +438,25 @@
                                               HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
                                               Content="{TemplateBinding Content}" />
 
-                            <ContentControl x:Name="PART_ActionIconPresenter"
-                                            Grid.RowSpan="2"
-                                            Grid.Column="3"
-                                            Width="{ThemeResource SettingsCardActionButtonWidth}"
-                                            Height="{ThemeResource SettingsCardActionButtonHeight}"
-                                            Margin="4,0,-8,0"
-                                            HorizontalAlignment="Center"
-                                            VerticalAlignment="Center"
-                                            HorizontalContentAlignment="Center"
-                                            VerticalContentAlignment="Center"
-                                            win:AutomationProperties.LocalizedControlType="button"
-                                            win:AutomationProperties.Name="{TemplateBinding ActionIconToolTip}"
-                                            win:HighContrastAdjustment="None"
-                                            CornerRadius="{ThemeResource ControlCornerRadius}"
-                                            FocusVisualMargin="-3"
-                                            FontFamily="{ThemeResource SymbolThemeFontFamily}"
-                                            IsTabStop="True"
-                                            ToolTipService.ToolTip="{TemplateBinding ActionIconToolTip}"
-                                            UseSystemFocusVisuals="True"
-                                            Visibility="Collapsed">
-                                <Viewbox MaxWidth="{ThemeResource SettingsCardActionIconMaxSize}"
-                                         MaxHeight="{ThemeResource SettingsCardActionIconMaxSize}">
-                                    <ContentPresenter win:HighContrastAdjustment="None"
-                                                      Content="{TemplateBinding ActionIcon}" />
-                                </Viewbox>
-                            </ContentControl>
+                            <Viewbox x:Name="PART_ActionIconPresenterHolder"
+                                     Grid.RowSpan="2"
+                                     Grid.Column="3"
+                                     Width="32"
+                                     Height="32"
+                                     MaxWidth="{ThemeResource SettingsCardActionIconMaxSize}"
+                                     MaxHeight="{ThemeResource SettingsCardActionIconMaxSize}"
+                                     Margin="{ThemeResource SettingsCardActionIconMargin}"
+                                     HorizontalAlignment="Center"
```

Comment:
Are any of these changes going to be a problem for folks updating? Would we see folks wanting to change the size here off 32x32? (smaller or bigger?)

This is the only part I'm trying to understand the nuances in of the PR in terms of ramifications of inverting the ViewBox/Content here.

Is this icon smaller than it used to be now? We don't have many icon examples in our sample app, so it's hard to tell.

![image](https://user-images.githubusercontent.com/24302614/231270339-c909e200-8c27-4de9-913c-1abfdf9a3901.png)

The icon I think is just different on Win10 as well...

Reply:
Oh yes, the `Height` and `Width` values are redundant here. Without those, it renders as expected on W11 (and changing the `SettingsCardActionIconMaxSize` still works to make icons smaller/bigger):
![image](https://user-images.githubusercontent.com/9866362/231496226-8b264177-b8d6-4ab1-a696-aa9844cf2f5d.png)